tests/active-active-rhel7-proxy/variables.tf

```diff
@@ -18,6 +18,21 @@ variable "acm_certificate_arn" {
   description = "The ARN of an existing ACM certificate."
 }
 
+variable "ca_certificate_secret_name" {
+  type        = string
+  description = "The secrets manager secret name of the Base64 encoded CA certificate."
+}
+
+variable "ca_private_key_secret_name" {
+  type        = string
+  description = "The secrets manager secret name of the Base64 encoded CA private key."
+}
+
+variable "domain_name" {
+  type        = string
+  description = "Domain for creating the Terraform Enterprise subdomain on."
```

Comment:
I think it would be beneficial to include an example here. From reading it I think this means that if you provide `staging.example.org`, then we'll create `tfe.staging.example.org`. But I had to read it a few times to make sure.

Reply:
Thanks for the feedback! I'll include it in the next PR.